.travis.yml

```diff
@@ -1,10 +1,8 @@
 sudo: false
 language: node_js
 node_js:
-  - "5"
-  - "4"
-  - "iojs"
-  - "0.12"
-  - "0.10"
+  - "10"
```

Comment:
It looks like the union of sinon's and assume's tested Node.js versions includes Node.js 8. It might make sense to test that here.